src/groups/mqb/mqbc/mqbc_storageutil.cpp

```diff
@@ -2518,6 +2528,10 @@ void StorageUtil::registerQueue(
         return;  // RETURN
     }
 
+    BALL_LOG_INFO << clusterDescription << " PartitionId [" << partitionId
```

Comment:
`PartitionId` -> `Partition`

Reply:
This results in double logging of queue registration together with Line 2427. Ideally, we print such log once in each code path.